docs/rum_getting_started.md

```diff
@@ -285,7 +285,9 @@ public class SampleApplication extends Application {
 {{< /tabs >}}
 {{< /site-region >}}
 
-The credentials for initialization require your application's variant name and uses the value of `BuildConfig.FLAVOR` or an empty string if you don't have variants. This enables the correct ProGuard `mapping.txt` file to automatically upload at build time so you can view de-obfuscated RUM error stack traces. For more information, see the [guide to uploading Android source mapping files][8].
+The initialization credentials require your application's variant name and uses the value of `BuildConfig.FLAVOR`. If you do not have variants, the credentials uses an empty string. 
+
+This enables the appropriate ProGuard `mapping.txt` file to automatically upload at build time so you can view deobfuscated RUM error stack traces. For more information, see the [Track Android Errors][8].
```

Comment:
Nope, the upload is performed by the Gradle plugin, not the SDK. 
Adding the variant actually allows us to match the errors reported from the app with the mapping files uploaded by the gradle plugin

Reply:
```suggestion
The Gradle plugin automatically uploads the appropriate ProGuard `mapping.txt` file at build time so you can view deobfuscated RUM error stack traces. For more information, see the [Track Android Errors][8].
```